superwall/src/main/java/com/superwall/sdk/Superwall.kt

```diff
@@ -382,8 +400,15 @@ class Superwall(
     /**
      * Do not use this function, this is for internal use only.
      */
-    fun setPlatformWrapper(wrapper: String) {
+    fun setPlatformWrapper(
+        wrapper: String,
+        version: String,
+    ) {
         dependencyContainer.deviceHelper.platformWrapper = wrapper
+        dependencyContainer.deviceHelper.platformWrapperVersion = version
+        ioScope.launch {
+            track(dependencyContainer.makeConfigAttributes())
+        }
```

Comment:
For this I track the device attributes rather than the config attributes, just because I've put the platform wrapper in the device attributes. Whereas I see you're also adding it to the config attributes. It probably doesn't matter, but maybe good to keep it consistent with ios unless you've got a strong opinion on it being in config attributes instead?

Reply:
Not really strong one, looking at it makes more sense since we got the app version et al there. Updated 👍